docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -9,80 +9,151 @@ track: "USERS › APPLICATION DEVELOPER › FOUNDATIONAL"
-* By itself, a Deployment can't provide ingress. The simplest way to configure ingress is to [create a Service](/docs/tasks/access-application-cluster/service-access-application-cluster/) of type `NodePort`, which takes care of routing external traffic to your cluster over a dynamically allocated IP address and port. The service also takes care of load balancing.
-* Optionally, learn more about providing external access:
-    * [More about Deployments](/docs/concepts/workloads/controllers/deployment/).
-    * [More about Services](/docs/concepts/services-networking/service/).
-    * [Learn more about Ingress](/docs/concepts/services-networking/ingress/).
+   Minikube includes a Docker daemon, but to create and push {% glossary_tooltip text="containers" term_id="container" %} as part of your development workflow, you'll want an independent Docker instance. Note that version 1.12 is recommended for full compatibility with Kubernetes, but a few other versions are tested and known to work.
 
-You'll also want to think about storage. Kubernetes provides different types of storage for different storage needs:
 
-* A Volume lets you define storage for your cluster that is tied to the lifecycle of a Pod. It is therefore more persistent than container storage. Learn [how to configure volume storage](/docs/tasks/configure-pod-container/configure-volume-storage/), or [read more about about volume storage](/docs/concepts/storage/volumes/).
-* A PersistentVolume and PersistentVolumeClaim let you define storage at the cluster level. Typically a cluster administrator defines the PersistentVolume objects for the cluster, and cluster users (application developers, you) define the PersistentVolumeClaim objects that your application requires. Learn [how to set up persistent storage for your cluster](/docs/tasks/configure-pod-container/configure-persistent-volume-storage/) or [read more about persistent volumes](/docs/concepts/storage/persistent-volumes/).
+You can get basic information about your cluster with the command `kubectl get nodes`. To get a good idea of what's really going on, however, you need to deploy an application to your cluster. This is covered in the next major section.
+
+## Deploy an application
+
+#### A Basic Workload
+
+The following examples demonstrate how to deploy basic apps to Kubernetes:
+  * [Deploy a simple, stateless nginx server](/docs/tasks/run-application/run-stateless-application-deployment/){:target="_blank"}. Nginx is often used as the web server to help provide ingress from outside your cluster (in other words, external requests).
+
+  * [Deploy a stateful, externally accessible MySQL database](/docs/tasks/run-application/run-single-instance-stateful-application/){:target="_blank"}. This introduces the concept of storage.
+
+Through these deployment tasks, you'll gain familiarity with the following concepts:
+* Fundamentals
```

Comment:
This introduction seems to say that by doing the suggested tasks, you will gain familiarity with configuration files, Pods, Deployment, Ingress, and Service.

But I don't see any use of an Ingress object in the suggested tasks.